node/blockchain/inner_models/block_message/node_declaration.py

```diff
@@ -15,6 +16,10 @@ class NodeDeclarationBlockMessage(BlockMessage):
     def make_block_message_update(
         cls, request: NodeDeclarationSignedChangeRequest, blockchain_facade: BlockchainFacade
     ) -> BlockMessageUpdate:
-        # TODO(dmu) CRITICAL: Implement this method
-        #                     https://thenewboston.atlassian.net/browse/BC-77
-        raise NotImplementedError()
+        message = request.message
+        account_state = AccountState(
+            account_lock=message.make_hash(),
```

Comment:
@buckyroberts should we update `account_lock` for node declaration blocks?

Reply:
Yep, let's update it for any signed change request made by that account. That way it prevents the network from processing the same signed change request twice since when the account lock is updated (which is part of the signed message) it also requires an updated digital signature.